Overview.src.html

```diff
@@ -889,11 +906,10 @@ <h4 id=terminology-headers>Headers</h4>
 <p>A <dfn>potential-navigation-or-subresource request</dfn> is a
 <span title=concept-request>request</span> whose
 <span title=concept-request-destination>destination</span> is
-"<code title>unknown</code>".
+"<code title>object</code>" or "<code title>embed</code>".
 
 <p>A <dfn>non-subresource request</dfn> is a <span title=concept-request>request</span>
-whose <span title=concept-request-destination>destination</span> is neither
-"<code title>subresource</code>" nor "<code title>unknown</code>".
+whose <span title=concept-request-destination>destination</span> is "<code title>document</code>", "<code title>serviceworker</code>", "<code title>sharedworker</code>", "<code title>worker</code>", or the empty string.
```

Comment:
I use 100 column forced wrap (keeping inline elements on a single line). I can take care of that I suppose while merging, but you're welcome to do it too.